js_modules/dagit/packages/core/src/instigation/LiveTickTimeline.tsx

```diff
@@ -136,19 +136,20 @@ export const LiveTickTimeline: React.FC<{
               return '';
             }
             const tick = ticks[tooltipItem.dataIndex];
+            const cursorLabel = tick.cursor ? `Cursor: ${tick.cursor}\n` : '';
             if (tick.status === InstigationTickStatus.SKIPPED && tick.skipReason) {
-              return tick.skipReason;
+              return cursorLabel ? [tick.skipReason, cursorLabel] : tick.skipReason;
```

Comment:
I know it's outside the scope of this PR, but it might be nice to add a comment here explaining what the array return value does. I guess each element is a line within the label?